src/app/app-routing.module.ts

```diff
@@ -3,7 +3,7 @@ import { NgModule } from '@angular/core';
 import { MainCoreLegacyComponent } from '@components/main-core-legacy/main-core-legacy-component';
 import { MainComponentLegacyComponent } from '@components/main-legacy/main-legacy.component';
 import { LoginPageComponent } from '@components/login/login-page-component';
-import { LogOutContainerComponent } from '@components/login/login-component/log-out-container';
+import { DotLogOutContainerComponent } from '@components/login/dot-login-component/dot-log-out-container';
```

Comment:
I'm in @freddyucv's branch and when I sen a one char string in the user field I get an error:

<img width="602" alt="Screen Shot 2019-03-19 at 9 36 47 AM" src="https://user-images.githubusercontent.com/751424/54619668-9aefcf00-4a2a-11e9-98aa-389b86768f20.png">